modules/juce_graphics/native/juce_mac_CoreGraphicsHelpers.h

```diff
@@ -69,6 +69,9 @@ CGContextRef juce_getImageContext (const Image&);
  void juce_convertColourGradientToDisplayColourSpace (
      ColourGradient& gradient, const void* rawNSScreenPointer);
 
+Image juce_returnImageWithNativeColourSpace (const Image& src,

```

Comment:
Why not `convert` here as well? Some of the other conversion functions already return the converted result so this wouldn't be a precedent

Reply:
`src` is redundant